bot-example.env

```diff
@@ -44,6 +44,11 @@ SENSITIVE_ROLES=
 # Separated by ;
 OPT_CATEGORIES=
 
-# A list of explicitly opt-blacklisted channel IDs .
+# A list of explicitly opt-blacklisted channel IDs.
 # Separated by ;
-OPT_BLACKLIST=
\ No newline at end of file
+OPT_BLACKLIST=
+
+# A list of category IDs in which NSA is disabled.
+# Separated by ;
+NSA_CATEGORIES=
```

Comment:
Unintuitive name; please change to something like NSA_BLACKLIST (or to be more politically correct, NSA_DENYLIST).

Reply:
I've changed the name to NSA_DENYLIST.